web/server.js

```diff
@@ -25,7 +25,13 @@ const broadcast = data => {
 
   _.each(
     wss.clients,
-    client => client.send(JSON.stringify(data))
+    client => {
+      try {
+        client.send(JSON.stringify(data);
```

Comment:
hm )

Reply:
that's a fix for #1582, not the best please to put it I admit ;)

Reply:
Ah you mean I forgot the `)`, good point, adding it back now!